### PR TITLE
Feature lwdev 7358 add loaders

### PR DESCRIPTION
**Pull request by Georg3MK**

### CODE REVIEW

File: src/components/Modal/styles.ts [hunk 1]
```diff
@@ -226,10 +226,10 @@ export const ModalWrapper = styled(Modal)`
   width: ${rem(300)};
 `;
 
-export const KycAndFundsBack = styled.div`
+export const CoveringBack = styled.div`
```
Comment (sergeyschegolev) [inline src/components/Modal/styles.ts]: We already have Backdrop Component(see Backdrop.tsx). I think we should leave only one of this 2 components(I mean CoveringBack or Backdrop)

File: src/components/Terminal/Terminal.tsx [hunk 2]
```diff
@@ -91,7 +92,8 @@ class Terminal extends React.Component<TerminalProps, {}> {
         splitPercentage: MAX_RIGHT_PADDING
       },
       splitPercentage: MAX_LEFT_PADDING
-    }
+    },
+    canBeShown: false
```
Comment (optimusway) [inline src/components/Terminal/Terminal.tsx]: let us rename it to `hasAccess`

File: src/components/Loader/commonLoader.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
```
Comment (optimusway) [inline src/components/Loader/commonLoader.tsx]: filename should be capitalized, i.e. `CommonLoader`

Comment (optimusway) [reply src/components/Loader/commonLoader.tsx]: i also have some suggestions to improve the whole PR

**Pull request merged**